tools/GridPROTEUS.py

```diff
@@ -473,8 +476,9 @@ def _thread_target(cfg_path):
     # Define parameter grid
     # -----
 
-    cfg_base = os.path.join(os.getenv('PROTEUS_DIR'),"input","t1c.cfg")
-    symlink  = "/network/group/aopp/planetary/RTP035_NICHOLLS_PROTEUS/outputs/t1c_v3"
+    cfg_base = os.path.join(os.getenv('PROTEUS_DIR'),"input","t1c.toml")
+    # symlink  = "/network/group/aopp/planetary/RTP035_NICHOLLS_PROTEUS/outputs/t1c_v3"
+    symlink = "/dataserver/users/formingworlds/nicholls/model_outputs/t1c_v4"
```

Comment:
Should this be a bit more generic in `main`?

Reply:
I agree. Made an issue for this https://github.com/FormingWorlds/PROTEUS/issues/162